crates/core/component/stake/src/component/validator_handler/validator_manager.rs

```diff
@@ -70,16 +70,16 @@ use {
 ///  │   ┌─────────────┐        ┌────────────┐     │              │◀──┘ │
 ///  └──▶│   Jailed    │◀───────│   Active   │◀───▶│   Inactive   │     │
 ///      └─────────────┘        └────────────┘     │              │◀────┘
-///             │                                  └──────────────┘      
-///             │                                          ▲             
-///             └──────────────────────────────────────────┘             
-///                                                                      
```

Comment:
thanks for doing this sort of thing while you are in the neighborhood. scout's rule :fleur_de_lis: